### PR TITLE
Membership API

### PR DESCRIPTION
**Pull request by ManojNathIC**

### CODE REVIEW

File: src/teacher/dto/teacher.dto.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class TeacherDto {
     description: "The subjectId of the teacher",
   })
   @Expose()
-  subjectIds: [String];
+  subjectIds: string;
```
Comment (coolbung) [inline src/teacher/dto/teacher.dto.ts:98]: Why did we change this ?

Comment (ManojNathIC) [reply src/teacher/dto/teacher.dto.ts:98]: Now I've solved this pushing new changes

Comment (ManojNathIC) [reply src/teacher/dto/teacher.dto.ts]: @coolbung sir please review and merge

File: src/group/group.controller.ts [hunk 2]
```diff
@@ -81,4 +88,28 @@ export class GroupController {
   ) {
     return await this.service.searchGroup(request, groupSearchDto);
   }
+
+  @Post(":id/members")
```
Comment (coolbung) [inline src/group/group.controller.ts:92]: This needs to be a GET API

File: src/group/group.controller.ts [hunk 3]
```diff
@@ -81,4 +88,28 @@ export class GroupController {
...
+  @ApiForbiddenResponse({ description: "Forbidden" })
+  public async findMembersOfGroup(
+    @Param("id") id: string,
+    @Query("role") role: string,
+    @Req() request: Request
+  ) {
+    return await this.membershipService.findMembersOfGroup(id, role, request);
+  }
+
+  @Post("/:userId")
```
Comment (coolbung) [inline src/group/group.controller.ts:104]: This needs to be `GET group/memberships/:userId`

**Pull request merged**